Source/utils/sdl2_to_1_2_backports.h

```diff
@@ -22,6 +22,11 @@
 
 #define SDL_MAX_UINT32 ((Uint32)0xFFFFFFFFu)
 
+#ifdef __PSP__
+// PSP has the wrong definition on SDL_config.h, so we use ours
+#define SDL_snprintf devilution::snprintf

```

Comment:
It'd be best to fix this in the PSP SDL instead

Reply:
its in draft mode so i think some hacks are expected for now :)